docs/apache-airflow-providers-cncf-kubernetes/operators.rst

```diff
@@ -152,6 +152,10 @@ XCom values differently than other operators. In order to pass a XCom value
 from your Pod you must specify the ``do_xcom_push`` as ``True``. This will create a sidecar container that runs
 alongside the Pod. The Pod must write the XCom value into this location at the ``/airflow/xcom/return.json`` path.
 
+.. note::
+  An invalid json content will fail, example ``echo 'hello' > /airflow/xcom/return.json`` fail and  ``echo '\"hello\"' > /airflow/xcom/return.json`` work
```

Comment:
May I know why is '"hello"' an valid json content?

Reply:
See the specs: https://www.json.org/json-en.html -> any value on it's own (string) is perfectly valid json. Also array is. What you **think** about json is the "object" part of it where you can define complex objects by combining different value types in more complex structures. But each "value" separately is also a valid json. Strings in JSON must be `"` quoted, so `hello` is not a valid value in json, but `"hello"` is. Also for example `23` is valid json (numeric value).

Reply:
From the specs:

> A value can be a string in double quotes, or a number, or true or false or null, or an object or an array. These structures can be nested.

Reply:
Got it. Thanks! Yep, I thought only objects and arrays are acceptable.

Reply:
It's a recursive definition - so every sub-tree and leaf is also a json :) ..

BTW. Little known fact (only loosely related). JSON is a 100% compatible subset of YAML. Every JSON is also a valid YAML file, and you can use YAML parser to load JSON file. I know it's surprising but it's true.